scripts/common/common.sh

```diff
@@ -621,8 +613,14 @@ function kubeconfig_setup_outro() {
 
     printf "To access the cluster with kubectl, copy kubeconfig to your home directory:\n"
     printf "\n"
+    log "Copying the kubeconfig to your home directory, with the following steps:"
+    printf "\n"
     printf "${GREEN}    cp "$(${K8S_DISTRO}_get_kubeconfig)" ~/.kube/config${NC}\n"
     printf "${GREEN}    chown -R ${owner} ~/.kube${NC}\n"
```

Comment:
this 100% needs to be reworded
I am too tired to do it well right now though

Reply:
I need to re-duo all tests as well and ensure that I could fix.
No worry, no rush we do that with time.

Reply:
100 % reworded. I think it is fine so I am solving this one.